Doc/library/datetime.rst

```diff
@@ -1398,13 +1397,13 @@ Instance methods:
 .. method:: datetime.utctimetuple()
 
    If :class:`.datetime` instance *d* is naive, this is the same as
-   ``d.timetuple()`` except that :attr:`~.time.struct_time.tm_isdst` is forced to 0 regardless of what
+   ``d.timetuple()`` except that :attr:`~.time.struct_time.tm_isdst` is forced to ``0`` regardless of what
    ``d.dst()`` returns. DST is never in effect for a UTC time.
 
    If *d* is aware, *d* is normalized to UTC time, by subtracting
    ``d.utcoffset()``, and a :class:`time.struct_time` for the
-   normalized time is returned. :attr:`!tm_isdst` is forced to 0. Note
-   that an :exc:`OverflowError` may be raised if *d*.year was
+   normalized time is returned. :attr:`!tm_isdst` is forced to ``0``. Note
+   that an :exc:`OverflowError` may be raised if ``d.year`` was
```

Comment:
Would not be better to use ``` :samp:`{d}.year` ```? And in other similar cases.

Reply:
I think double backquotes is better in this case, but I don't feel strongly about it.